packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/__tests__/quanticGeneratedAnswerContent.test.js

```diff
@@ -122,4 +133,19 @@ describe('c-quantic-generated-answer-content', () => {
       );
     });
   });
+
+  describe('the reactivity of the answerContentFormat property', () => {
+    it('should load the markdown dependencies when the answerContentFormat property is changed to text/markdown', async () => {
```

Comment:
```suggestion
  describe('the answerContentFormat property', () => {
    it('should load the markdown dependencies when set to text/markdown, async () => {
```

Reply:
I wanna highlight the fact that this is about reactivity and not about just passing a public. property.